test/end-to-end/pages/tasklist/tasklist.js

```diff
@@ -1,24 +1,16 @@
-/* eslint-disable no-await-in-loop */
 'use strict';
-const content = require('app/resources/en/translation/tasklist');
+
+const contentEn = require('app/resources/en/translation/tasklist');
+const contentCy = require('app/resources/cy/translation/tasklist');
 const testConfig = require('config');
+const pageUnderTest = require('app/steps/ui/tasklist');
 
```

Comment:
This is undoing performance work - including these files takes time - do we need to?